cluster/gce/gci/configure.sh

```diff
@@ -115,19 +115,13 @@ function split-commas {
 }
 
 function install-gci-mounter-tools {
-    local -r rkt_version="v1.18.0"
-    local -r gci_mounter_version="v2"
-    local -r rkt_binary_sha1="75fc8f29c79bc9e505f3e7f6e8fadf2425c21967"
-    local -r rkt_stage1_fly_sha1="474df5a1f934960ba669b360ab713d0a54283091"
-    local -r gci_mounter_sha1="851e841d8640d6a05e64e22c493f5ac3c4cba561"
-    download-or-bust "${rkt_binary_sha1}" "https://storage.googleapis.com/kubernetes-release/rkt/${rkt_version}/rkt"
-    download-or-bust "${rkt_stage1_fly_sha1}" "https://storage.googleapis.com/kubernetes-release/rkt/${rkt_version}/stage1-fly.aci"
-    download-or-bust "${gci_mounter_sha1}" "https://storage.googleapis.com/kubernetes-release/gci-mounter/gci-mounter-${gci_mounter_version}.aci"
-    local -r rkt_dst="${KUBE_HOME}/bin/"
-    mv "${KUBE_HOME}/rkt" "${rkt_dst}/rkt"
-    mv "${KUBE_HOME}/stage1-fly.aci" "${rkt_dst}/stage1-fly.aci"
-    mv "${KUBE_HOME}/gci-mounter-${gci_mounter_version}.aci" "${rkt_dst}/gci-mounter-${gci_mounter_version}.aci"
-    chmod a+x "${rkt_dst}/rkt"
+    mkdir "${CONTAINERIZED_MOUNTS_HOME}"
+    chmod a+x "${CONTAINERIZED_MOUNTS_HOME}"
+    local -r mounter_sha="8003b798cf33c7f91320cd6ee5cec4fa22244571"
+    download-or-bust "${mounter_sha}" "https://storage.googleapis.com/kubernetes-release/gci-mounter/mounter.tar"
+    mv "${KUBE_HOME}/mounter.tar" /tmp/mounter.tar
```

Comment:
I wanted to move it to tmpfs as it won't be needed later on. 
Now that I think more of it, I can delete this once it has been extracted. Thoughts?